File: trail1/홀수로 이루어진 정사각형/square-with-odd-numbers.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int[][] board = new int[N][N];
        for (int i=0; i<N; i++) {
            for (int j=0; j<N; j++) {
                board[i][j] = 11 + 2*i + 2*j;
            }
        }
        for (int i=0; i<N; i++) {
            for (int j=0; j<N; j++) {
                System.out.print(board[i][j] + " ");
            }
            System.out.println();
        }
    }
}

File: trail1/홀수로 이루어진 정사각형/square-with-odd-numbers.c
#include <stdio.h>

int main() {
    int N;
    scanf("%d", &N);
    int board[N][N];
    for (int i=0; i<N; i++) {
        for (int j=0; j<N; j++) {
            board[i][j] = 11 + 2*i + 2*j;
        }
    }
    for (int i=0; i<N; i++) {
        for (int j=0; j<N; j++) {
            printf("%d ", board[i][j]);
        }
        printf("\n");
    }
    return 0;
}